packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/edit-display-name/controller.js

```diff
@@ -0,0 +1,62 @@
+export default class EditNameCtrl {
+  /* @ngInject */
+  constructor($translate, Nutanix) {
+    this.$translate = $translate;
+    this.Nutanix = Nutanix;
+  }
+
+  onUpdateDisplayNameFormSubmit() {
+    if (!this.actionEnabled) {
+      return this.goBack();
+    }
+
+    this.loading.update = true;
+
+    return this.Nutanix.updateDisplayName({
```

Comment:
You are not updating Nutanix cluster name here. You are updating Nutanix node name which is dedicated server. Please reuse DedicatedService. Lets have a discussion on this.